spec/support/connect_to_mediaflux.rb

```diff
@@ -1,36 +1,32 @@
 # frozen_string_literal: true
 
-  def reset_mediaflux_root
-
-      # Clean out the namespace before running tests to avoid collisions
-      user = User.new
-      destroy_request = Mediaflux::NamespaceDestroyRequest.new(
-        session_token: user.mediaflux_session,
-        namespace: Rails.configuration.mediaflux[:api_root_to_clean]
-      )
-      destroy_request.destroy
-      if destroy_request.error?
-        puts "Error destroying the mediaflux root namespace #{destroy_request.response_error}" # allow the message to show in CI output
-      end
-
-      # then create it and the project root collection and namespace so it exists for any tests
-      create_test_root_namespace(user)
-      ProjectMediaflux.create_root_ns(session_id: user.mediaflux_session)
-      project_parent = Mediaflux::Connection.root_collection
-      ProjectMediaflux.prepare_parent_collection(project_parent:, session_id: user.mediaflux_session)
+def reset_mediaflux_root
+  # Clean out the namespace before running tests to avoid collisions
+  user = User.new
+  destroy_request = Mediaflux::NamespaceDestroyRequest.new(
+    session_token: user.mediaflux_session,
+    namespace: Rails.configuration.mediaflux[:api_root_to_clean]
+  )
+  destroy_request.destroy
+  if destroy_request.error?
+    puts "Error destroying the mediaflux root namespace #{destroy_request.response_error}" # allow the message to show in CI output
   end
 
-  def create_test_root_namespace(user)
-    create_request = Mediaflux::NamespaceCreateRequest.new(
-      session_token: user.mediaflux_session,
-      namespace: Rails.configuration.mediaflux[:api_root_to_clean]
-    )
-    create_request.resolve
-    if create_request.error?
-      puts "Error creating #{Rails.configuration.mediaflux[:api_root_to_clean]}: #{create_request.response_error}" # allow the message to show in CI output
-    end
-  end
+  # then create it and the project root collection and namespace so it exists for any tests
+  create_test_root_namespace(user)
+  ProjectMediaflux.create_root_tree(session_id: user.mediaflux_session)
```

Comment:
Rubocop reformated this file so the changes look daunting, but the only change really is to call `ProjectMediaflux.create_root_tree` instead of `ProjectMediaflux.create_root_ns` and `ProjectMediaflux.prepare_parent_collection`